src/app/admin/institution-settings/overseer-images/overseer-image-list.component.html

```diff
@@ -6,43 +6,42 @@ <h3>Overseer Images</h3>
     </div>
   </div>
   <table mat-table [dataSource]="dataSource" matSort (matSortChange)="sortTableData($event)" class="mat-elevation-z3">
-
     <!-- Name Column -->
     <ng-container [formGroup]="formData" matColumnDef="name">
-      <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+      <th mat-header-cell *matHeaderCellDef mat-sort-header>OnTrack Name</th>
```

Comment:
We cant use "OnTrack" as that is not what it is called everywhere. `DoubtfireConstants` has an `ExternalName` that you can use instead.